go/stellar/remote/remote.go

```diff
@@ -695,54 +1094,39 @@ type isMobileResult struct {
 }
```

Comment:
i don't think `isMobileResult` is used anymore

Reply:
correct. removed.